### ISSUE TITLE
Does cornerstone support avi format?

### ISSUE THREAD

**Issue by yimingluci** [cornerstonejs/cornerstone#293]

**Comment by swederik** [comment]
No, I guess you'd want the HTML video tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video

What is the use case?

**Comment by yimingluci** [comment]
@swederik , basically my clients have some video files in avi format. They want to upload the avi files and convert them into muti-frame dicom format, and then view them and put annotations on them.
